lib/generator._js

```diff
@@ -264,10 +265,19 @@ ScriptGenerator.prototype.generateAspNetCoreDeploymentScript = function (_) {
     throw new Error('Only batch script files are supported for ASP.NET Core Application');
   }
 
+  if (!this.solutionPath && this.useMSBuild) {
+    throw new Error('Missing solution file path (--solutionFile). Solution file path has to be specified when used with --use-msbuild');
```

Comment:
Is the converse also true? i.e. is it valid to pass solution when *not* passing `--use-msbuild`?